src/Compute/Compute/Generated/DedicatedHost/DedicatedHostUpdateMethod.cs

```diff
@@ -123,6 +131,11 @@ public override void ExecuteCmdlet()
             Mandatory = false)]
         public DedicatedHostLicenseTypes LicenseType { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "Redeploy the dedicated host. The operation will complete successfully once the dedicated host has migrated to a new node and is running. To determine the health of VMs deployed on the dedicated host after the redeploy check the Resource Health Center in the Azure Portal. Please refer to https://docs.microsoft.com/azure/service-health/resource-health-overview for more details.")]
+        public bool Redeploy { get; set; }
```

Comment:
Refer to this [guide](https://github.com/Azure/azure-powershell/blob/674123b1e883dc764cfef1cb728e54f54ef9eab3/documentation/development-docs/design-guidelines/parameter-best-practices.md#bool-vs-switchparameter). Please use "SwitchParameter" instead.